starter/config/index-head-config.js

```diff
@@ -52,6 +53,6 @@ module.exports = {
 		// <!-- You can customize the default -->
 		{ name: "theme-color", content: "#0076c8" },
 		{ name: "msapplication-TileColor", content: "#0076c8" },
-		{ name: "msapplication-TileImage", content: "assets/images/app-icons/ms-icon-144x144.png" }
+		{ name: "msapplication-TileImage", content: "assets/images/app-icons/ms-icon-144x144.png", "=content": true }
```

Comment:
Same remark: Can you add a comment to mention why we should add the `"=content": true` ?